src/index.js

```diff
@@ -3,3 +3,19 @@ import Scores from './leaderboard';
 
 const leaderboard = new Scores();
 leaderboard.displayScores();
+// refresh btn
+document.querySelector('.refresh').addEventListener('click', () => {
+  leaderboard.displayScores();
+});
+// form submission
+const form = document.querySelector('.add-score');
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const name = form.elements.name.value;
+  const score = form.elements.score.value;
+  leaderboard.addParticipant(name, score).then((res) => {
+    if (!res.error) {
+      form.reset();
+    }
+  });
+});
```

Comment:
Data is not being stored and retrieved from the API as intended. Kindly ensure that:
- [ ] The submit button should hit the POST API endpoint and post user entries.
- [ ] When the Refresh button is clicked, it should hit the GET API endpoint causing the scores list to change.

Reply:
all resolved ✅ . Thanks a lot 🤓

Reply:
This issue is well resolved! ✔️